README.md

```diff
@@ -76,6 +73,13 @@ Wakaama supports additional client related options. These are only available if
  
 Please note: LwM2M version 1.0 is only supported by clients, while servers are backward compatible.
 
+### CoaP Settings
+
+- WAKAAMA_RAW_BLOCK1_REQUESTS For low memory client devices where it is not possible to keep a large post or put request in memory to be parsed (typically a firmware write).
```

Comment:
Should we name this ` WAKAAMA_COAP_RAW_BLOCK1_REQUESTS`? Block transfer is a CoAP feature.

Reply:
Good idea. This makes the variable name quite long. But it's more consistent with the other variables.